app/client/cli/system.go

```diff
@@ -0,0 +1,81 @@
+package cli
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/pokt-network/pocket/rpc"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	systemCmd := NewSystemCommand()
+	rootCmd.AddCommand(systemCmd)
+}
+
+func NewSystemCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "System",
+		Short:   "Commands related to health and troubleshooting of the node instance",
+		Aliases: []string{"sys"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(systemCommands()...)
+
+	return cmd
+}
+
+func systemCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Health",
+			Short:   "RPC endpoint liveness",
+			Long:    "Performs a simple liveness check on the node RPC endpoint",
+			Aliases: []string{"health"},
+			RunE: func(cmd *cobra.Command, args []string) error {
+
+				client, err := rpc.NewClientWithResponses(remoteCLIURL)
+				if err != nil {
+					return nil
+				}
+				response, err := client.GetV1HealthWithResponse(cmd.Context())
+				if err != nil {
+					return unableToConnectToRpc(err)
+				}
+				statusCode := response.StatusCode()
+				if statusCode == http.StatusOK {
+					fmt.Printf("✅ RPC reporting healthy status for node @ \033[1m%s\033[0m\n\n%s", remoteCLIURL, response.Body)
```

Comment:
Is the `\033[1m%` in place for color? If so, can you add a tiny comment saying `URL is color X`

Reply:
https://github.com/pokt-network/pocket/pull/169/commits/f2c765fa18a3e0fd02134907db7f537dc1fc591e